turbopack/crates/turbopack-core/src/resolve/mod.rs

```diff
@@ -1502,14 +1464,14 @@ fn merge_results_with_affecting_sources(
 
 #[turbo_tasks::function]
 pub async fn resolve_raw(
-    lookup_dir: FileSystemPath,
+    lookup_path: FileSystemPath,
```

Comment:
AFAICT, this really is a directory path and not a file path. Which is easy to get wrong

Reply:
most of the `lookup_path` parameters are all directories not files....

i was gonna  rename them all, but then decided it was too invasive and it looks like i over corrected and renamed this one too. 

Fixed